doc-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
+#rpm-python
```

Comment:
The intention is to list all dependencies with comments why these don't apply to the use case. The bottom line is that when installing rebase-helper or packaging it for a distribution, you need to add these dependencies. When removed, the information will be scattered in multiple places. I would prefer to have the list complete and this is the reason why I included it with the comment. Unless you have a really strong reason why to remove this, I would like to keep it in the file. It is harmless and provides additional information.